performance-tests/TestAppPlain/android/app/src/main/java/com/testappplain/MainApplication.kt

```diff
@@ -0,0 +1,38 @@
+package com.testappplain
+
+import android.app.Application
+import com.facebook.react.PackageList
+import com.facebook.react.ReactApplication
+import com.facebook.react.ReactHost
+import com.facebook.react.ReactNativeApplicationEntryPoint.loadReactNative
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import com.facebook.react.defaults.DefaultReactHost.getDefaultReactHost
+import com.facebook.react.defaults.DefaultReactNativeHost
+
+class MainApplication : Application(), ReactApplication {
```

Comment:
This should fix the linter complain
```suggestion
class MainApplication :
    Application(),
    ReactApplication {
```